tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -375,6 +375,49 @@ Search Items In Resources Section
     Sleep   5
     Input Text  xpath://input[@class="pf-c-search-input__text-input"]       ${element}
 
+Open Element By Name
+    [Documentation]     It opens the element by enabling Quick Start filter in Resources Section of ODS Dashboard.
+    [Arguments]     ${element}
+    Click Link      Resources
+    Sleep   5
+    Select Checkbox     //input[@id="quickstart--check-box"]
+    Wait Until Page Contains Element    //article[@id="${element}"]
+    Click Link          //article[@id="${element}"]//a
+    Wait Until Page Contains ELement    //div[@class="pf-c-drawer__panel-main"]     5
+
+Get The Count Of Description
+    [Documentation]     Gives the count of description which is shown while starting the tour.
+    ${count}=   Get ELement Count   //div[@class="pfext-quick-start-content"]/ul/li
+    Click Button    //button[@data-test="Start button"]
+    Wait Until Page Contains Element    //div[@class="pfext-quick-start-content"]
+    [Return]    ${count}
+
+Verify The Progress Of The Items
+    [Documentation]     Verifies the label on the item card which shows the progress and closes the sidebar.
+    [Arguments]     ${element}  ${progress_string}
+    Wait Until Page Contains Element    //button[@data-testid="qs-drawer-close"]    5
+    Click Button    //button[@data-testid="qs-drawer-close"]
+    Page Should Not Contain Element     //div[@class="pf-c-drawer__panel-main"]
```

Comment:
I think these shouldn't be part of this kw. This kw should just check the status. Are these 3 lines closing the QuickStart? then maybe those need to be in a dedicated kw and used outside of this kw

Reply:
Yes I have created **Close The Sidebar By Clicking Leave** keyword which closes the sidebar by clicking on leave button and have separated from **Verify The Progress Of Item** Keyword.